### PR TITLE
kubetail 1.6.13 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/kubetail.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Kubetail < Formula
+  desc "Bash script to tail Kubernetes logs from multiple pods at the same time"
+  homepage "https://github.com/johanhaleby/kubetail"
+  url "https://github.com/johanhaleby/kubetail/archive/1.6.13.tar.gz"
+  sha256 "a4aeea1cffede44d5e8b030b6ead3ffe25fd3c1e7c5b7c82905df65cceac1254"
+  license "Apache-2.0"
+  head "https://github.com/johanhaleby/kubetail.git", branch: "master"
+
+  depends_on "kubernetes-cli" => :test
```
Comment (SMillerDev) [inline Formula/kubetail.rb:9]: How is it only in the test and not at runtime?

File: Formula/kubetail.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class Kubetail < Formula
+  desc "Bash script to tail Kubernetes logs from multiple pods at the same time"
```
Comment (SMillerDev) [inline Formula/kubetail.rb:2]: Shouldn't this have a `uses_from_macos "bash"` then?

File: Formula/kubetail.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    bin.install "kubetail"
+
+    bash_completion.install "completion/kubetail.bash" => "kubetail"
+    zsh_completion.install "completion/kubetail.zsh" => "_kubetail"
+    fish_completion.install "completion/kubetail.fish"
+  end
+
+  test do
+    assert_match "server localhost:8080 was refused",
+      shell_output("#{bin}/kubetail brew-pod-v1 2>&1", 1)
```
Comment (SMillerDev) [inline Formula/kubetail.rb:21]: Can we tell it to use a specific port? This is a pretty common port to be in use.

File: Formula/kubetail.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/kubetail.rb:9]: My thinking is this is the CLI, and users can install the k8s client in their preferred way.

File: Formula/kubetail.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/kubetail.rb:2]: good point.

File: Formula/kubetail.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/kubetail.rb:9]: Will it work without a k8 client?

**Pull request closed without merging**